enn_ppo/enn_ppo/train.py

```diff
@@ -201,8 +201,8 @@ def train(args: argparse.Namespace) -> float:
     env_cls = ENV_REGISTRY[args.gym_id]
     # env setup
     envs = EnvList([env_cls() for _ in range(args.num_envs)])
-    obs_space = env_cls.obs_space()
-    action_space = env_cls.action_space()
+    obs_space = envs.envs[0].obs_space()
```

Comment:
This looks like an operation that we would do most of the times with an `EnvList` so I would create both `obs_space` and `action_space` methods in `EnvList`